### PR TITLE
#163483766 Create articles with price

### PR DESCRIPTION
**Pull request by wombolo**
#### What does this PR do?
Ensure all articles, created and updated have price tags

#### Description of Task to be completed?
All articles should have price tags

#### How should this be manually tested?
After installation of dependencies, run `sequelize db:migrate` to update databases.
Then try creating an article with price field(which must be a decimal figure with 2 d.p.).
Example:

`POST > http://localhost:9001/api/v1/arts` VALUES:
```
title:Cirrhosis of the Sky,
description:Cirrhosis of the Sky, The Syntax Podcasts isn't suitable for all ages,
categoryId:1,
price:5000.00,
media:[{"url":"https://farm3.staticflickr.com/2817/33968464326_a6f9cbc754_k","extension":"jpg"},   {"url":"https://images.pexels.com/photos/1047540/pexels-photo-1047540.jpeg?dl&fit=crop&crop=entropy&w=1280&h=853","extension":"jpeg"}]
```

#### Any background context you want to provide?
Only artists are able to create articles

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163483766

#### Questions:

### CODE REVIEW

**Review by abejide001 — APPROVED** [review]
LGTM

File: src/controllers/ArtsController.js [hunk 1]
```diff
@@ -141,7 +145,7 @@ class ArtsController {
 
       if (artistId !== artToUpdate.artistId) {
         const response = new Response(
-          'Not Ok',
+          'Forbidden',
           403,
           'Unauthorized to Edit Article',
           {}
```
Comment (daniellamarr) [inline src/controllers/ArtsController.js:151]: I don't think passing an empty object here is quite necessary on this error response. Why not remove the empty object.

Comment (wombolo) [reply src/controllers/ArtsController.js:151]: Ok thanks

**Review by julietezekwe — APPROVED** [review]
Looks ok though there are many file changes not relating to the task. please take note of that.

**Review by Proception — APPROVED** [review]
Looks Good To Me

**Pull request closed without merging**